resources/meilisearch-release.md

```diff
@@ -78,32 +78,40 @@ The steps are the same as for `How to create the first pre-release (or RC i.e. R
 
 ### How to do the official release
 
-1. Go the the GitHub interface, in the [`Release` section](https://github.com/meilisearch/meilisearch/releases).
+1. Copy/paste this TODO steps on your side
+<details>
+<summary>Release todo</summary>
+- [ ] Launch the release on the GitHub interface</br>
+- [ ] Check binaries and Docker images are ready: check the assets are attached to the release + run the Docker image on your machine to ensure it starts correctly (at least)</br>
+- [ ] Check the <a href="https://github.com/Homebrew/homebrew-core/pulls">Homebrew PR</a> has been created.</br>
+- [ ] Once everything is deployed, make a message on #ms-release to announce the new release: ping integration-team and docs-team since they should deploy their own scope now.</br>
+- [ ] Bring back commits on the `release-vX.Y.Z` into `main` (cf <a href="https://github.com/meilisearch/engine-team/blob/main/resources/meilisearch-release.md#after-the-release">this section</a>) of the guide)</br>
```

Comment:
```suggestion
- [ ] Check the <a href="https://github.com/Homebrew/homebrew-core/pulls">Homebrew PR</a> has been created.</br>
- [ ] Once everything is deployed, make a message on #ms-release to announce the new release: ping integration-team and docs-team since they should deploy their own scope now.</br>
- [ ] Bring back commits on the `release-vX.Y.Z` into `main` (cf <a href="https://github.com/meilisearch/engine-team/blob/main/resources/meilisearch-release.md#after-the-release">this section</a>)</br>
```